uint8 id

geometry_msgs/Point position
geometry_msgs/Vector3 relative_velocity

float32 length
float32 width
float32 orientation_angle

int8 dynamic_property
int8 class_type
